client/src/components/pages/sign-in/password-field/index.tsx

```diff
@@ -0,0 +1,47 @@
+// TODO: remove it and use common PasswordField component
+
+import React, { useCallback, useState, FC } from 'react';
+import { FieldAttributes } from 'formik';
+import { combineClasses } from 'helpers/combineClasses.helper';
+import FormField, { FORM_FIELD_CLASS } from '../form-field';
+import styles from './password-input.module.scss';
+import hideIcon from 'assets/icons/hide.svg';
+import showIcon from 'assets/icons/show.svg';
+
+const PasswordField: FC<FieldAttributes<any>> = (props) => {
+	const [isHiding, setHiding] = useState(true);
+	const inputRef = React.createRef<HTMLButtonElement>();
```

Comment:
Yot should use [useRef hook](https://medium.com/trabe/react-useref-hook-b6c9d39e2022) in functional components.